### DISCUSSION TITLE
Add udp2raw implementation to bypass censoring on Wireguard protocol

### DISCUSSION THREAD

**Discussion by scropion86** [wg-easy/wg-easy#1098] [Ideas]
**Describe the bug**

It is not a bug it is more as feature request to Add udp2raw implementation to bypass censoring on Wireguard protocol
which is implemented in many places either by ISP or government.

**Comment by pheiduck** [comment]
I think it wouldn't make sense to mitigate the security of WireGuard's data traffic. As it could possibly harm the user through attacks etc.
